examples/plot_nonstationary_error.py

```diff
@@ -0,0 +1,205 @@
+"""
+Non-stationarity of elevation measurement errors
```

Comment:
After reading through, I believe this should be a section in the xdem documentation, rather than hidden in the example gallery?

Reply:
Right now, I've parted things this way:
**Section** = Background + Equations + One-liner example of functionality pointing to example
**Example** = Fully commented workflow using the one-liner
With one example per one-liner command referenced in the documentation sections.

We can move things around later, it should be quite easy once the content is there!